<launch>

  <node pkg="laserScan_Similarity" type="laserScan_Similarity_Rings_Continuous" name="laserScan_Similarity_Rings_Continuous" output="screen" >

    <param name="inputQueueSize" value="1"/>
    <param name="rangeThreshold" value="0.0"/>
    <param name="isVoxelFilter" value="false"/>
    <param name="voxelFilterConfig" value="$(find laserScan_Similarity)/cfg/voxel_filter.yaml" />

    <param name="sectionNumOfR" value="20"/>
    <param name="minRange" value="0.0"/>
    <param name="maxRange" value="100.0" />

    <param name="sectionNumOfI" value="20"/>
    <param name="minIntensity" value="0.0"/>
    <param name="maxIntensity" value="100.0"/>
    
    <param name="durationSecond" value="3"/>
    <param name="velodyneRings" value="16"/>
    <param name="upStart" value="0"/>

  </node>
</launch>
